Documentation/GeneralConventions/FileStructure.rst

```diff
@@ -71,41 +71,13 @@ displayed on GitHub and GitLab.
 .. _index-rst:
 .. _start-file:
 
-Startpage: :file:`Documentation/Index.rst`
-------------------------------------------
+Start page: :file:`Documentation/Index.rst`
+-------------------------------------------
 
 The documentation index file at :file:`Documentation/Index.rst` is the starting
```

Comment:
Maybe better would be: "The documentation start page is the entry point". "Index file" sounds pretty technical.